resume.html

```diff
@@ -0,0 +1,345 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW"
+      crossorigin="anonymous"
+    ></script>
+    <!-- Google Font Quicksand and Poppins -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Add a link to your custom CSS here -->
+    <link href="./css/mainresume.css" rel="stylesheet" />
+    <link href="./css/body.css" rel="stylesheet" />
+    <link href="./css/navbar.css" rel="stylesheet" />
+
+    <title>Resume</title>
+  </head>
+
+  <body class="body">
+    <header class="container-fluid">
+      <div class="row">
+        <nav
+          class="navbar navbar-expand-md fixed-top"
+          style="background-color: white; height: 5rem;"
+        >
+        <a class="col-6 col-md-5 navbar-brand" href="#">

```

Comment:
Since `row` sets `display: flex` it needs the `col-...` elements to be its direct children in order to work properly. This means you should always follow `row` with `col-...` (here your nav is in between the two, breaking the direct relationship)